spring-boot/admin/src/main/java/eu/coatrack/admin/controllers/AdminController.java

```diff
@@ -81,570 +85,571 @@
-        ApiKey newApiKey = apiKeysForTestService.get(apiKeysForTestService.size() - 1);
-
-        // get the URL(s) for the test service proxy(s)
-        Map<String, List<String>> proxyURLPerNewApiKey = new TreeMap<>();
-        List<String> proxiesUrlList = proxyRepository.customSearchForAllProxiesForGivenServiceApiId(newApiKey.getServiceApi().getId())
-                .stream()
-                .filter(proxy -> proxy.getPublicUrl() != null)
-                .filter(proxy -> proxy.getPublicUrl() != "")
-                .map(Proxy::getPublicUrl)
-                .collect(Collectors.toList());
-
-        proxyURLPerNewApiKey.putIfAbsent(newApiKey.getKeyValue(), proxiesUrlList);
-
-        model.addAttribute("testService", gettingStartedTestService);
-        model.addAttribute("proxiesPerApiKey", proxyURLPerNewApiKey);
-        model.addAttribute("apiKeys", newApiKey);
-
-        // decide about which fragment to return
-        String returnedFragment = (whichFragmentToLoad.equals("table"))
-                // returns the ApiKeyTable Fragment
-                ? "admin/fragments/api-keys/consumer/list :: apiKeyTable"
-                // returns the Url Fragment
-                : "admin/fragments/consumer_wizard/wizard :: gatewayCallURL";
-
-        return returnedFragment;
-    }
-
-    private ServiceApi loadTestServiceForConsumerWizard() {
-        //Getting the test service
-        log.debug("trying to load test service for provider '{}' and uriIdentifier '{}'",
-                gettingStartedTestServiceProvider,
-                gettingStartedTestServiceIdentifier);
-        ServiceApi testServiceApi = serviceApiRepository.findServiceApiByServiceOwnerAndUriIdentifier(
-                gettingStartedTestServiceProvider,
-                gettingStartedTestServiceIdentifier);
-        log.debug("test service loaded: {}", testServiceApi);
-        return testServiceApi;
-    }
+						// temporalFirstTimeFlag = false;
+						mav.setViewName(ADMIN_STARTPAGE);
+
+					} else {
+						// IT IS A CONSUMER USER
+						mav.setViewName(ADMIN_CONSUMER_HOME_VIEW);
+					}
+
+				}
+
+			} else {
+
+				// The user is new for our database therefore we try to retrieve as much user
+				// info is possible from Github
+				OAuth2AuthenticationDetails details = (OAuth2AuthenticationDetails) auth.getDetails();
+
+				RestTemplate restTemplate = new RestTemplate();
+
+				HttpHeaders headers = new HttpHeaders();
+				headers.add("Authorization", "token " + details.getTokenValue());
+				HttpEntity<String> githubRequest = new HttpEntity<String>(headers);
+
+				ResponseEntity<String> userInfoResponse = restTemplate.exchange(GITHUB_API_USER, HttpMethod.GET,
+						githubRequest, String.class);
+				String userInfo = userInfoResponse.getBody();
+				// String userInfo = restTemplate.getForObject(GITHUB_API_USER +
```

Comment:
These comments which include code seems to me like old code. Do you think it is important to keep?
If not, please remove them

Reply:
the email parameter is passed by java arguments

Reply:
@perezdf I am just worrying that when we merge this branch, then the production will not work because a email is not configured correctly? So do we have a email configured for production??